agent/engine/docker_image_manager.go

```diff
@@ -273,6 +275,12 @@ func (imageManager *dockerImageManager) isImageOldEnough(imageState *image.Image
 	return ageOfImage > imageManager.minimumAgeBeforeDeletion
 }
 
+//TODO: change createdTime to pullAtTime when docker support it in the future
```

Comment:
I think it would be more reasonable in **TODO** that change createdTime to image last used time since this is what we really want to keep track of.